core/util/src/main/scala/net/liftweb/util/TimeHelpers.scala

```diff
@@ -427,18 +424,29 @@ trait TimeHelpers { self: ControlHelpers =>
       case e: Exception => logger.debug("Error parsing date "+in, e); Failure("Bad date: "+in, Full(e), Empty)
     }
   }
+
+  implicit class PeriodExtension[P <% Period](period: P) {
+    def later: DateTime = new DateTime(millis).plus(period)
+
+    def ago: DateTime = new DateTime(millis).minus(period)
+  }
+
 }
 
+@deprecated("TimeSpan to Date/DateTime conversion will be removed for possibility of mistakes in on-duration operations")
 trait ConvertableToDate {
+  @deprecated("TimeSpan to Date conversion will be removed for possibility of mistakes in on-duration operations")
   def toDate: Date
+  @deprecated("TimeSpan to DateTime conversion will be removed for possibility of mistakes in on-duration operations")
   def toDateTime: DateTime
   def millis: Long
 }
 
+@deprecated("TimeSpan to Date/DateTime conversion will be removed for possibility of mistakes in on-duration operations")
 object ConvertableToDate {
+  @deprecated("TimeSpan to Date conversion will be removed for possibility of mistakes in on-duration operations")
   implicit def toDate(in: ConvertableToDate): Date = in.toDate
+  @deprecated("TimeSpan to DateTime conversion will be removed for possibility of mistakes in on-duration operations")
   implicit def toDateTime(in: ConvertableToDate): DateTime = in.toDateTime
   implicit def toMillis(in: ConvertableToDate): Long = in.millis
```

Comment:
I agree with it. The only one type conversion that exists now in TimeHelpers is joda Duration -> TimeSpan conversion. What do you think about deprecation of it?
Other implicit conversions are for class extensions.